package.json

```diff
@@ -90,6 +91,8 @@
     "postcss-import": "^12.0.1",
     "postcss-modules": "^2.0.0",
     "prettier": "^2.0.5",
+    "react-hooks-testing-library": "^0.6.0",
+    "react-test-renderer": "^16.13.1",
```

Comment:
I feel like the wrong packages are being installed. I can't find any of these imports in this pull-requests. But I do find a new `@testing-library/react-hooks`.

So I guess that one should have been installed instead of these two?